src/python/WMComponent/DBS3Buffer/DBSBufferUtil.py

```diff
@@ -202,3 +204,38 @@ def getCompletedWorkflows(self):
         result = wfCompletedDAO.execute(transaction = False)
 
         return result
+    
+    def getPhEDExDBSStatusForCompletedWorkflows(self, summary=False):
+        
+        """
+        _gettPhEDxDBSStatuForCompletedWorkflows_
+
+        """
+        wfCompletedStatusDAO = self.daoFactory(classname = "CheckStatusForCompletedWorkflows")
+        result = wfCompletedStatusDAO.execute(transaction = False)
+        if summary:
+            result = self.summaryPhEDExDBSStatus(result)
+            
+        return result
+
+    def summaryPhEDExDBSStatus(self, data):
+        """
+        """
+        summary = defaultdict(dict)
+        for workflow, value in data.iteritems():
+            # only getting completed workflows
+            summary[workflow]["Completed"] = True
+            
+            if value["NotInPhEDEx"] == 0 and value["InPhEDEx"] > 0:
```

Comment:
or should it be `if value["NotInPhEDEx"] == 0 and value["InPhEDEx"] >= 0:`, for cases that no output files were produced? One could say the phedex injection is done (though there were no files :D). ... I don't know, it all depends on how you interpret it.
Same below for DBS